.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,10 +12,11 @@ A clear and concise description of what the bug is.
 
 **To Reproduce**
 Steps to reproduce the behavior:
+
 1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
+1. Click on '....'
+1. Scroll down to '....'
+1. See error
 
```

Comment:
Numbering seems off

Reply:
This is a valid ordered list in markdown. See [this](https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md029---ordered-list-item-prefix). In fact, this prevents the cognitive load of renumbering if we need to insert new item in the ordered list.